### DISCUSSION TITLE
So if I want to block ads on adult sites (i.e., allowing the adult sites)...?

### DISCUSSION THREAD

**Discussion by iconoclasthero** [StevenBlack/hosts#2628] [Q&A]
I assume to block ads from adult sites I wish to browse (e.g., porn, gambling, weed, booze) that in and of themselves are on blocklists for various reasons (e.g., one presumably doesn't want one's kid watching porn, gambling, or getting f-ed up), I do **not** want to use the "porn" extension of the hosts list...but are there lists aimed at reducing the unwanted content while intending to browse adult sites?

I don't know how clear that is so...
**tl;dr:**
Goal: Browse adult sites (e.g., booze, weed, gambling, porn) that are otherwise on blocklists themselves with fewer ads.
Question: Are there lists specifically to browse adult sites with fewer ads?

**Comment by StevenBlack** [comment]
Thank you for your question @iconoclasthero.

We offer 31 different hosts lists.

The one you want is the basic list, the one on the repo home page, whose readme is part of the repo home page.

Here is the full list of lists we have. The `basic` is on the home page, and the others are subfolders in the `alternates/` subfolder.

```
basic
fakenews (includes basic)
fakenews-gambling (includes basic)
fakenews-gambling-only
fakenews-gambling-porn (includes basic)
fakenews-gambling-porn-only
fakenews-gambling-porn-social (includes basic)
fakenews-gambling-porn-social-only
fakenews-gambling-social (includes basic)
fakenews-gambling-social-only
fakenews-only
fakenews-porn (includes basic)
fakenews-porn-only
fakenews-porn-social (includes basic)
fakenews-porn-social-only
fakenews-social (includes basic)
fakenews-social-only
gambling (includes basic)
gambling-only
gambling-porn (includes basic)
gambling-porn-only
gambling-porn-social (includes basic)
gambling-porn-social-only
gambling-social (includes basic)
gambling-social-only
porn (includes basic)
porn-only
porn-social (includes basic)
porn-social-only
social (includes basic)
social-only
```

**Comment by iconoclasthero** [comment]
Thanks.

Having looked around a bit, I saw [this](https://github.com/mhxion/pornaway/blob/master/hosts/porn_ads.txt) too:
```
# PornAway blacklist
# D̲i̲r̲e̲c̲t̲ ̲P̲o̲r̲n̲ ̲A̲D̲S̲ ̲F̲i̲l̲e̲ (see line 13, 14)
⋮
# Additional source & thanks to: Clefspeare13 (github), Sinfonietta (github).
# This list is all of  porn ADS. Enabling it will not block porn sites but only the porn ads. 
# There's a different host file available for verified porn sites only.
⋮
```

I'll have to diff them to see how they compare I guess.

**Comment by StevenBlack** [comment]
@iconoclasthero I think you may be confused.

What you have there is a porn list.

You should pick one of our lists that doesn't block porn.
